swan-lake/integration-tutorials/sending-a-message-to-a-service.md

```diff
@@ -58,18 +58,21 @@ Follow the instructions given in this section to develop the service.
     The generated record type will be as follows.
 
     ```ballerina
-    type Doctor record {|
+    type Doctor record {
         string name;
         string hospital;
         string category;
         string availability;
         decimal fee;
-    |};
+    };
     ```
 
     > **Note:**
     > While it is possible to work with the JSON payload directly, using record types offers several advantages including enhanced type safety, data validation, and better tooling experience (e.g., completion).
 
+    > **Note:**
+    > Opted for a [closed record](https://ballerina.io/learn/by-example/records/) in the source code to emphasize the type-safety and maintainability.
```

Comment:
```suggestion
    > When the fields of the JSON objects are expected to be exactly those specified in the sample payload, the generated records can be updated to be [closed records](https://ballerina.io/learn/by-example/controlling-openness/), which would indicate that no other fields are allowed or expected.
```

Reply:
I feel like, The link you have provided mainly tells that making a record an open record to allow other fields. But shouldn't we emphasize more on making the records closed?

Reply:
No, right? The first sentence `Use record {| ... |} to describe a record type that allows exclusively what is specified in the body.` and the first example show exactly closed records.

Reply:
As a second thought, It tells both open and closed records. Let's put this. +1